src/stream.js

```diff
@@ -21,12 +22,12 @@ export const recurseReaderAsEvent = async (reader, emitter) => {
 
   if (!result.done) {
     return recurseReaderAsEvent(reader, emitter);
-  } else {
-    emitter.emit('done');
   }
+
+  emitter.emit('done');
 };
 
-export const stream = (url, options) => {
+export default (url, options) => {
```

Comment:
Should a name be provided to this function for better stack traces?

Reply:
It will be named by the import from the importer.